src/game/potato_game_sup.erl

```diff
@@ -0,0 +1,38 @@
+%% @doc potato_game_sup supervisor manages individual game instances
+%%
+%% Each game instance is added using add_game(Verifiers)
+%%
+%% This has the code that responds to block requests.
+%% Current idea is that only verifiers should do that,
+%% though it is not inconcievable that players can also
+%% provide that information to each other.
+-module(potato_game_sup).
+-behaviour(supervisor).
+
+-export([start_link/0, init/1, add_game/2]).
+
+-include_lib("stdlib/include/assert.hrl").
+
+start_link() ->
+  %% this will register with name potato_game_sup I think?
+  supervisor:start_link({local, ?MODULE}, ?MODULE, []).
+
+init([]) ->
+  io:format("starting potato_game_sup~n"),
+  {ok, {{simple_one_for_one, 1, 5},[
```

Comment:
What's simple_one_for_one?

Reply:
I think only 1 type of child allowed and you can add as many of that child. With the other types, the children are static and you can refer to them via their id